packages/query/src/bitcoin/ordinals/inscriptions-infinite.query.ts

```diff
@@ -0,0 +1,161 @@
+import { useCallback } from 'react';
+
+import { HDKey } from '@scure/bip32';
+import { useInfiniteQuery } from '@tanstack/react-query';
+
+import { getTaprootAddress } from '@leather.io/bitcoin';
+import { createNumArrayOfRange } from '@leather.io/utils';
+
+import { useLeatherNetwork } from '../../leather-query-provider';
+import { QueryPrefixes } from '../../query-prefixes';
+import { useBestInSlotApiRateLimiter } from '../../rate-limiter/best-in-slot-limiter';
+import { useBitcoinClient } from '../clients/bitcoin-client';
+
+const stopSearchAfterNumberAddressesWithoutOrdinals = 5;
+const addressesSimultaneousFetchLimit = 5;
+
+const inscriptionsLazyLoadLimit = 20;
+
+export function useBestInSlotGetInscriptionsInfiniteQuery({
```

Comment:
I've added sep request, as "hiro" one is used in some other hooks, like e.g. `useNumberOfInscriptionsOnUtxo`
I'll refactor it separately, bc `useNumberOfInscriptionsOnUtxo` doesn't work properly and actually shouldn't use infinite query